src/pages/Schedule/SchedulePage.scss

```diff
@@ -1,28 +0,0 @@
-.schedule-page {
```

Comment:
Removed `SchedulePage` is no longer used.
https://github.com/sparkletown/sparkle/pull/1302/commits/a6cd04b7351b3b004d58205991ae7fd7e65d4ead